config/locales/en.yml

```diff
@@ -1287,14 +1281,23 @@ en:
       steps_applicant_personal_details_form:
         <<: *PERSONAL_DETAILS_FIELDS
         previous_name: Enter your previous name
+        has_previous_name_options:
```

Comment:
Better if we do the same as we did with gender, leave it in the dictionary but for now have an old and a new version so it continues working on the forms not yet migrated.